.github/workflows/manual-deploy-obscuro-gateway-database.yml

```diff
@@ -0,0 +1,120 @@
+
+
+      - name: 'Login via Azure CLI'
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      # This will fail some deletions due to resource dependencies ( ie. you must first delete the vm before deleting the disk)
+      - name: 'Delete deployed VMs'
+        uses: azure/CLI@v1
+        with:
+          inlineScript: |
+            $(az resource list --tag ${{env.RESOURCE_TAG_NAME}}=true --query '[]."id"' -o tsv | xargs -n1 az resource delete --verbose -g Testnet --ids) || true
+
+      # This will clean up any lingering dependencies - might fail if there are no resources to cleanup
+      - name: 'Delete VMs dependencies'
+        uses: azure/CLI@v1
+        with:
+          inlineScript: |
+            $(az resource list --tag ${{env.RESOURCE_TAG_NAME}}=true --query '[]."id"' -o tsv | xargs -n1 az resource delete --verbose -g Testnet --ids) || true
+
+
+      - name: 'Create VM for Gateway database node on Azure'
+        uses: azure/CLI@v1
+        with:
+          inlineScript: |
+            az vm create -g Testnet -n "${{env.RESOURCE_STARTING_NAME}}-OG-MariaDB-${{ GITHUB.RUN_NUMBER }}" \
+            --admin-username obscurouser --admin-password "${{ secrets.OBSCURO_NODE_VM_PWD }}" \
+            --public-ip-address-dns-name "obscurogateway-mariadb-${{env.RESOURCE_STARTING_NAME}}" \
+            --tags deploygroup=ObscuroGateway-mariaDB-${{env.RESOURCE_STARTING_NAME}}-${{ GITHUB.RUN_NUMBER }}  ${{env.RESOURCE_TAG_NAME}}=true \
+            --size 	Standard_D4_v5 --image Canonical:0001-com-ubuntu-server-focal:20_04-lts-gen2:latest \
+            --public-ip-sku Basic --authentication-type password
+
+      - name: 'Open Obscuro Gateway db ports on Azure'
+        uses: azure/CLI@v1
+        with:
+          inlineScript: |
+            az vm open-port -g Testnet -n "${{env.RESOURCE_STARTING_NAME}}-OG-MariaDB-${{ GITHUB.RUN_NUMBER }}" --port 3306
+
+        # To overcome issues with critical VM resources being unavailable, we need to wait for the VM to be ready
+      - name: 'Allow time for VM initialization'
+        shell: bash
+        run: sleep 30
+
+      - name: 'Start Obscuro gateway on Azure'
+        uses: azure/CLI@v1
+        with:
+          inlineScript: |
+            az vm run-command invoke -g Testnet -n "${{env.RESOURCE_STARTING_NAME}}-OG-MariaDB-${{ GITHUB.RUN_NUMBER }}"  \
+            --command-id RunShellScript \
+            --scripts 'mkdir -p /home/obscuro \
+            && sudo apt-get update \
+            && sudo apt-get install -y gcc \
+            && sudo snap refresh && sudo snap install --channel=1.18 go --classic \
+            && curl -fsSL https://get.docker.com -o get-docker.sh && sh ./get-docker.sh \
+            && git clone --depth 1 -b ${{ env.BRANCH_NAME }} https://github.com/obscuronet/go-obscuro.git /home/obscuro/go-obscuro \
+            && docker network create --driver bridge node_network || true \
```

Comment:
`|| true ` suppresses any errors. Is there any purpose for doing it?
I guess if any of the steps fail the deployment will not be sucessful

Reply:
Good point. 
It does suppress errors but only for that if clause, so if other clauses fail then the command fails. 

We want to forcibly create a node_network so that if we do create more containers they can intra-container communicate between them. 

It's more of a standard, everywhere that we create docker containers we create the `node-network` (it then guarantees that all containers can communicate between them). Not sure if it's worth removing. Will remove it if you feel strongly about it.